foodsaving/cases/templates/conflict_resolution_case_continued.mjml

```diff
@@ -0,0 +1,30 @@
+<mjml>
+    <mj-head>
+        <mj-include path="../../base/templates/_base-head.mjml"/>
+        <mj-title>{% trans affected_user_name=affected_user.display_name|safe %}Continue the discussion about the conflict with {{ affected_user_name }}{% endtrans %}
+        </mj-title>
+    </mj-head>
+    <mj-body>
+        <mj-include path="../../base/templates/_header.mjml"/>
+        <mj-section background-color="#ffffff" padding="40px 0 40px 0">
+            <mj-column>
+                <mj-text>
+                    <h1>{% trans affected_user_name=affected_user.display_name|safe %}Continue the discussion about the conflict with {{ affected_user_name }}{% endtrans %}
+                    <p>{% trans %}The most acceptable outcome of the last voting round is to continue discussion and start another round of voting.{% endtrans %}</p>
```

Comment:
'The most acceptable outcome' sounds weird to me. I'd replace the phrase with 'The result of the last voting round is to continue discussion and start another round of voting.'